src/android/InAppBrowser.java

```diff
@@ -1046,8 +1046,6 @@ public void postMessage(String data) {
                     String databasePath = cordova.getActivity().getApplicationContext().getDir("inAppBrowserDB", Context.MODE_PRIVATE).getPath();
                     settings.setDatabasePath(databasePath);
                     settings.setDatabaseEnabled(true);
-                    settings.setAppCachePath(databasePath);
```

Comment:
What about previous versions of Android? This should have a if 33+ etc etc etc

Reply:
It doesn't even allow me to compile. 
![studio64_hjGHAqrqYT](https://github.com/j5int/cordova-plugin-inappbrowser/assets/25819871/1f11c660-4dc1-4332-97da-93c9328f28b8)

Reply:
Plus it has been removed upstream

Reply:
Ok cool - can we just test this on an early version of Android Simulator? whats our lowest testing point?

Reply:
Android-12 is making SDK 24 (Android 7) the minimum. Will setup a android 7 emulator and test our app. Will revert back

Reply:
Tested and I added some videos for android 7 and android 14 emulator. The inappbrowser loads normally and I can interact with it.